pegasus/migrations/83_create_channel_table_metadata.rb

```diff
@@ -0,0 +1,20 @@
+Sequel.migration do
+  up do
+    create_table?(:channel_table_metadata, charset: 'utf8') do
+      primary_key :id, unsigned: true, null: false
+      foreign_key :app_id, index: true, null: false
+      foreign_key :storage_id, index: true
+      String :table_name, size: 50, index: true, null: false
+
+      Text :column_list
+
+      DateTime :updated_at, null: false
+
+      index [:channel_id,:storage_id,:table_name], unique: true
+    end
```

Comment:
The presence of the `storage_id` column for this table may turn into bit of a land mine if we ever want to actually use `user-tables` for anything. The reason for this is that `user-tables` should share metadata (most notably next_id) across all users of the table, since that's the behavior they would currently have if we started using them today. If we have `storage_id` as a column and as part of the metadata_hash, then there will be no logical way to represent user-table metadata in this table in the future. Based on this, I would recommend using "table type" (shared or user) here instead of `storage_id`.